src/translations/dist/en.json

```diff
@@ -637,21 +801,30 @@
     "level_na": {
       "avalanche_levels": "not availabe"
     },
+    "Licenses": "Licenses",
     "lift_access": "Lift access",
     "lift_status": "Lift status",
     "lightning": {
       "event_types": "lightning"
     },
+    "link": "link",
+    "Links": "Links",
+    "List mode": "List mode",
     "List of versions for language:": "List of versions for language:",
+    "Load my preferences": "Load my preferences",
+    "loading": "loading in progress",
     "local_product": {
       "waypoint_types": "local product"
     },
     "locality": {
       "waypoint_types": "locality"
     },
     "Location / elevation / orientation": "Location / elevation / orientation",
-    "Log in to post the first comment": "Log in to post the first comment",
+    "Log in to post a comment": "Log in to post a comment",
```

Comment:
I don't find the `Log in to post a comment` key, but I still find the `Log in to post the first comment`..

Reply:
CommentsBox, line 47